content/fr/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,438 @@
+* Réseau de cluster : ensemble de liens, logiques ou physiques, facilitant la communication au sein d'un cluster selon le [modèle de réseau Kubernetes](/docs/concepts/cluster-administration/networking/).
+* Service : un Kubernetes [Service](/docs/concepts/services-networking/service/) identifiant un ensemble de pods à l'aide de sélecteurs d'étiquettes. Sauf indication contraire, les services sont supposés avoir des adresses IP virtuelles routables uniquement dans le réseau du cluster.
+
+## Qu'est-ce qu'un ingress ?
+
+Ingress (ou une entrée réseau), ajouté à Kubernetes v1.1, expose les routes HTTP et HTTPS de l'extérieur du cluster à
+{{<link text = "services" url = "/docs/concepts/services-networking/service/">}} au sein du cluster.
+Le routage du trafic est contrôlé par des règles définies sur la ressource Ingress.
+
+```none
+    internet
+        |
+   [ Ingress ]
+   --|-----|--
+   [ Services ]
+```
+
+Un Ingress peut être configuré pour donner aux services des URLs accessibles de l'extérieur, du trafic de charge équilibrée, la terminaison SSL/TLS et un hébergement virtuel basé sur le nom. Un [contrôleur d'Ingress](/docs/concepts/services-networking/ingress-controllers) est responsable de l'exécution de le l'Ingress, généralement avec un load-balancer (équilibreur de charge), bien qu'il puisse également configurer votre routeur périphérique ou des interfaces supplémentaires pour aider à gérer le trafic.
+
+Un Ingress n'expose pas de ports ni de protocoles arbitraires. Exposer des services autres que HTTP et HTTPS à Internet généralement utilise un service de type [Service.Type=NodePort](/docs/concepts/services-networking/service/#nodeport) ou [Service.Type=LoadBalancer](/docs/concepts/services-networking/service/#loadbalancer).
+
+## Conditions préalables
+
+{{<feature-state for_k8s_version = "v1.1" state = "beta">}}
+
+Avant de commencer à utiliser un Ingress, vous devez comprendre certaines choses. Un Ingress est une ressource bêta.
+
+{{< note >}}
+Vous devez avoir un [contrôleur d'Ingress](/docs/concepts/services-networking/ingress-controllers) pour lancer un Ingress. Seule la création d'une ressource Ingress n'a aucun effet.
+{{< /note >}}
+
+GCE/GKE (Google Cloud Engine / Google Kubernetes Engine) déploie un contrôleur d’Ingress sur le master (le maître de kubernetes). Revoir les [limitations bêta](https://github.com/kubernetes/ingress-gce/blob/master/BETA_LIMITATIONS.md#glbc-beta-limitations) de ce contrôleur si vous utilisez GCE/GKE.
+
+Dans les environnements autres que GCE/GKE, vous devrez peut-être [déployer un contrôleur d'Ingress](https://kubernetes.github.io/ingress-nginx/deploy/). Il y a un certain nombre de
+[contrôleurs d'Ingress](/docs/concepts/services-networking/ingress-controllers) parmi lesquels vous pouvez choisir.
+
+### Avant de commencer
+
+Dans l’idéal, tous les contrôleurs d’Ingress devraient correspondre à cette spécification, mais les divers contrôleurs sont légèrement différents.
```

Comment:
mais les divers contrôleurs sont légèrement différents. ? Peut être à revoir cette phrase, on parle des implémentation.

Reply:
Je propose :  ". Cependant le fonctionnement est légèrement différent d'un contrôleur à un autre (en fonction de son implémentation)."

Le texte d'origine est : "but the various ingress controllers operate slightly differently",  je trouve qu'on parle de : comment le contrôleur opère les différentes manipulations des ingresses.
La phrase entre () est optionnelle.

Qu'est ce que t'en penses ?

Reply:
Je suis d'accord avec la suggestion de @awaddiz